### DISCUSSION TITLE
Not able to download iraf-2.17 on Mac M2 chip with Ventura 13.1

### DISCUSSION THREAD

**Discussion by Kim4213** [iraf-community/iraf#254] [Q&A]
**Describe the bug**
I get the error below when trying to download iraf-2.17. I also tried to follow the steps on https://github.com/iraf-community/iraf/discussions/201 and started from scratch, but it did not help.

----------------------- GENERIC ------------------------
+ cd generic
+ sh -x mkpkg.sh
+ set -e
+ flex -o lexyy.c tok.l
+ gcc -c -g -Wall -O2 -I/Users/kimphan/Desktop/iraf-2.17/include generic.c yywrap.c lexyy.c
+ gcc generic.o lexyy.o yywrap.o /Users/kimphan/Desktop/iraf-2.17/unix/hlib/libboot.a /Users/kimphan/Desktop/iraf-2.17/lib/libsys.a /Users/kimphan/Desktop/iraf-2.17/lib/libvops.a /Users/kimphan/Desktop/iraf-2.17/unix/hlib/libos.a /Users/kimphan/Desktop/iraf-2.17/unix/bin.macos64/libf2c.a -lm -o generic.e
ld: file too small (length=8) file '/Users/kimphan/Desktop/iraf-2.17/lib/libvops.a' for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [sysgen] Error 1
kimphan@Kims-MacBook-Pro iraf-2.17 %

**To Reproduce**
Steps to reproduce the behavior.

 1. Download Xcode using xcode-select --install (currently Xcode 14.2, Build version 14C18)
 2. Download and open the tar file (https://iraf-community.github.io/install.html) on Desktop 
 3. Change directory to iraf-2.17 
 4. Execute install script using ./install 
 5. I use zshell and in my .zshrc file i wrote:  export PATH=/Users/kimphan/Desktop/iraf-2.17/bin/:$PATH,  export iraf=/Users/kimphan/Desktop/iraf-2.17/
 6. make macos64
 7. make sysgen 2>&1 | tee build.log 

**Expected behavior**
Download should be successful?

**System details:**
 - OS: macOS 13.1 Ventura
 - Version 2.17

**Comment by olebole** [comment]
Could you attach the generated log file (build.log)?

**Comment by Kim4213** [comment]
Here it is: 
[build.log](https://github.com/iraf-community/iraf/files/10411823/build.log)

**Comment by olebole** [comment]
The problem is the same as in #201; the IRAF-specific tools are not found during the build. There are lines like
```
xc -c -w access.x aread.x […]
zsh:1: command not found: xc
```
which show that the "xc" command was not found. This is probably because `/Users/kimphan/.iraf/bin/` is not in your **PATH**.
